dawn/src/dawn/Support/Twine.h

```diff
@@ -22,8 +22,10 @@
 #include "dawn/Support/SmallVector.h"
 #include "dawn/Support/StringRef.h"
 #include "dawn/Support/Unreachable.h"
+
 #include <cassert>
 #include <iosfwd>
+#include <ostream>
```

Comment:
necessary?

Reply:
Yeah, methods defined on lines 244 and 247 take std::ostreams.